### PR TITLE
Migrated all gradle files to kts

### PR DESCRIPTION
**Pull request by marinacoelho**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        vectorDrawables.useSupportLibrary = true
+    }
+
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        }
+    }
+    flavorDimensions.add("environment")
```
Comment (thatfiredev) [inline app/build.gradle.kts:35]: nit: You can use the `+=` operator instead.
```suggestion
    flavorDimensions.add("environment")
```

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
```
Comment (thatfiredev) [inline build.gradle.kts:6]: Do we still need this? I assume that now that it is declared in `settings.gradle.kts` it shouldn't be necessary.

**Review by thatfiredev — APPROVED** [review]
Two minor comments, but overall it looks great! :)

File: app/build.gradle.kts (hunk 1 above)
Comment (marinacoelho) [reply app/build.gradle.kts:35]: Done!

File: build.gradle.kts (hunk 2 above)
Comment (marinacoelho) [reply build.gradle.kts:6]: You're right, we don't need! Removed.

**Pull request merged**